lib/jefferies_tube/custom_prompts.irbrc.rb

```diff
@@ -0,0 +1,31 @@
+rails_env = JefferiesTube.configuration.environment
+
+if rails_env
+  color = "\e[0m" #Default to white text on no background
+  current_app = JefferiesTube.configuration.prompt_name
+
+  # shorten some common long environment names
+  if rails_env == "development"
```

Comment:
We have a couple other common env names: dev, staging, qa. I think we should include them too. Maybe just make them all blue?